### PR TITLE
Feat/ics export

### PR DESCRIPTION
**Pull request by svevang**
Targeting https://github.com/PRX/publish.prx.org/issues/586 this PR sets up a production calendar export for a series in the ical format.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -23,6 +23,9 @@ services:
     environment:
       VIRTUAL_HOST: cms.prx.docker
       LOCAL_ENV: "true"
+      FEEDER_DB_HOST: pg
```
Comment (cavis) [inline docker-compose.yml]: Are these really required here?  I thought they were only used by the feeder import script.  But maybe the tests need them or something.

File: app/representers/api/series_representer.rb [hunk 2]
```diff
@@ -9,9 +9,14 @@ class Api::SeriesRepresenter < Api::BaseRepresenter
   property :created_at, writeable: false
   property :updated_at, writeable: false
   property :app_version, writeable: false
+  property :calendar_ics_url, writeable: false, exec_context: :decorator
```
Comment (cavis) [inline app/representers/api/series_representer.rb]: Shouldn't this be a `_link`, not a property?

File: app/models/story.rb [hunk 3]
```diff
@@ -159,6 +159,13 @@ def self.paranoia_scope
 
   scope :public_stories, -> { published.network_visible.series_visible }
 
+  scope :public_calendar_stories, -> do
```
[the remark this answers is not in the document]
Comment (cavis) [reply app/models/story.rb]: Hmm... I'm wondering if that's a security issue.  Would there be any problem with just throwing the calendar under the `/authorization/series` endpoints?

File: docker-compose.yml (hunk 1 above)
Comment (svevang) [reply docker-compose.yml]: Yep,  the tests in docker wont run unless these are set.

File: app/models/story.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (cavis) [reply app/models/story.rb]: Update: does need to be a publicly accessible URL.  But should leave the story.title out, for security reasons.  Instead, use the Season/Episode number, falling back to the drop date.

File: app/representers/api/series_representer.rb (hunk 2 above)
Comment (svevang) [reply app/representers/api/series_representer.rb]: The gist is that you can't "follow" this link (e.g. using the PRXAccess gem), you can't inspect its attributes etc.. It's not a HAL doc. So it seems to more closely follow as an attribute of the model.

File: docker-compose.yml (hunk 1 above)
Comment (cavis) [reply docker-compose.yml]: +1

File: app/representers/api/series_representer.rb (hunk 2 above)
Comment (cavis) [reply app/representers/api/series_representer.rb]: Hmm... that feels like a shortcoming of the gem.  You should be able to get the URL for a link without following it.  How do we currently handle non-json `type`d links, like images?

```json
{
  "prx:image": {
    "id": 8457,
    "filename": "hermoney-3000x3000-768x768.jpg",
    "size": 109018,
    "_links": {
      "self": {
        "href": "/api/v1/accounts/217417/image",
        "profile": "http://meta.prx.org/model/image/account"
      },
      "profile": {
        "href": "http://meta.prx.org/model/image/account"
      },
      "enclosure": {
        "href": "https://cms.prx.org/pub/1c4c1a73d04f60aaac090731ac1c0c65/0/web/account_image/8457/medium/hermoney-3000x3000-768x768.jpg",
        "type": "image/jpeg"
      },
      "original": {
        "href": "https://cms.prx.org/pub/b56e9d6e928725337d987fa5c3d673f1/0/web/account_image/8457/original/hermoney-3000x3000-768x768.jpg",
        "type": "image/jpeg"
      }
    }
  }
}
```

Comment (cavis) [reply app/representers/api/series_representer.rb]: Or maybe I misunderstood your comment.  But in any case, having a non-HAL mime in the link `type` is valid per https://tools.ietf.org/html/draft-kelly-json-hal-06#section-5.3

File: app/services/public_calendar_title_placeholder_builder.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
+class PublicCalendarTitlePlaceholderBuilder
```
Comment (cavis) [inline app/services/public_calendar_title_placeholder_builder.rb]: Service, not a concern?  (I don't actually have a strong opinion)

**Pull request merged**